src/components/Button.js

```diff
@@ -1,8 +1,10 @@
 import { React } from 'react'
+import { rgba, darken } from 'polished'
 import { css } from '@emotion/core'
 import { Link } from 'gatsby'
+
+import colors from '../lib/colors'
```

Comment:
I think that we should pass via theme only the things that are variable and keep defined colors that are consts in separate file under the `lib/` folder